compiler/rustc_error_codes/src/error_codes.rs

```diff
@@ -454,6 +454,7 @@ E0768: include_str!("./error_codes/E0768.md"),
 E0769: include_str!("./error_codes/E0769.md"),
 E0770: include_str!("./error_codes/E0770.md"),
 E0771: include_str!("./error_codes/E0771.md"),
+E0773: include_str!("./error_codes/E0773.md"),
```

Comment:
Why did you skip E0772? An open PR is adding it too?

Reply:
E0772 already exists, it just doesn't have a long-form description: https://github.com/rust-lang/rust/blob/245ad9fdb69e2c25a2d31a2504d4a36cee17b10d/compiler/rustc_error_codes/src/error_codes.rs#L637